src/Chronos.php

```diff
@@ -88,18 +88,20 @@ class Chronos extends DateTimeImmutable implements ChronosInterface
      * for more on the possibility of this constructor returning a test instance.
      *
      * @param \DateTimeInterface|string|int|null $time Fixed or relative time
-     * @param \DateTimeZone|string|null $tz The timezone for the instance
+     * @param \DateTimeZone|string|null $timezone The timezone for the instance
      */
-    public function __construct(DateTimeInterface|string|int|null $time = 'now', DateTimeZone|string|null $tz = null)
-    {
+    public function __construct(
+        DateTimeInterface|string|int|null $time = 'now',
+        DateTimeZone|string|null $timezone = null
```

Comment:
Since time zone is 2 words, I think we should follow php use `timeZone` to be accurate.

Reply:
Hmm, php names the class TimeZone but the constructor parameter for DateTime is all lower case `timezone`? Unfortunate.

We won't be extending DateTime though so ...

Reply:
Yes, PHP capitalizes the `z` only for the class name. For arguments (`$timezone`) and methods names (`setTimezone()`) it's lowercased. I would prefer we keep the same even if we are not going to extend `DateTimeImmutable`.

Reply:
yea, let's follow it.